fc/mef.py

```diff
@@ -736,230 +506,223 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
     if plot_filename is None:
         plot_filename = str(data_beads)
```

Comment:
That's a fine reason to not use `infile`, then. Would still not use `__str__`; require caller to specify it if we cannot extract it reliably from `data_beads`.